substrait_consumer/consumers.py

```diff
@@ -26,21 +26,22 @@ def __init__(self, db_connection=None):
         self.db_connection.execute("INSTALL substrait")
         self.db_connection.execute("LOAD substrait")
 
-    def setup(self, db_connection, file_names: Iterable[str]):
+    def setup(self, db_connection, created_tables, file_names: Iterable[str]):
         self.db_connection = db_connection
+        self.load_tables_from_parquet(created_tables, file_names)
 
-    def run_substrait_query(self, substrait_query: bytes) -> pa.Table:
```

Comment:
Why are we changing this interface? Can we keep this as it is an may be in the caller we convert the `json` string to bytes?

Reply:
I changed it to be consistent since all the producers can create the json plan, but not all producers can produce the byte representation (isthmus cannot)